galsim/chromatic.py

```diff
@@ -1747,17 +1768,21 @@ def __init__(self, *args, **kwargs):
         if kwargs:
             raise TypeError("Got unexpected keyword argument(s): %s"%kwargs.keys())
 
+        # Accumulate convolutant .SEDs.
+        self.SED = args[0].SED
+        for obj in args[1:]:
+            self.SED *= obj.SED
```

Comment:
Sure.  Done.

This also made me realize that a GSObject being dimensionless is more of the rule than the exception, so I think I may go back and redocument this accordingly.  I.e., the only time GSObjects are dimensionful is in drawImage when exptime != 1 or area != 1.  At other times, e.g. when duck-typing as ChromaticObjects, they should be viewed as dimensionless.